app/src/main/java/org/p2p/wallet/sell/interactor/HistoryItemMapper.kt

```diff
@@ -33,10 +45,10 @@ class HistoryItemMapper(private val resources: Resources) {
                 }
             }
         }
-        return sellHistoryItems + rpcHistoryItems
+        historyItemFlow.tryEmit(sellHistoryItems + rpcHistoryItems)
```

Comment:
why tryEmit? I don't think plus with two lists can throw an exception

Reply:
tryEmit is async, and emit is suspend function. We should avoid using suspend, and start to using flow for data )